graphql_compiler/tests/integration_tests/integration_test_helpers.py

```diff
@@ -63,3 +64,43 @@ def compile_and_run_sql_query(schema, graphql_query, parameters, engine, metadat
             results.append(dict(result))
         trans.rollback()
     return results
+
+
+def compile_and_run_neo4j_query(schema, graphql_query, parameters, neo4j_client):
+    """Compile and run a Cypher query against the supplied graph client."""
+    compilation_result = compile_graphql_to_cypher(
+        schema, graphql_query, type_equivalence_hints=None)
+    query = compilation_result.query
+    with neo4j_client.driver.session() as session:
+        results = session.run(query, parameters)
+    return results.data()
+
+
+def compile_and_run_redisgraph_query(schema, graphql_query, parameters, redisgraph_client):
+    """Compile and run a Cypher query against the supplied graph client."""
+    converted_parameters = {
+        name: try_convert_decimal_to_string(value)
```

Comment:
There shouldn't be any `Decimal`-typed values in tests involving Cypher, no? I thought we decided that neither RedisGraph nor Neo4j supported such a data type.

Reply:
Ah, right-- this a leftover from before we found out about that. Will remove